src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -919,46 +907,54 @@ class A : Attribute
                 {
                     public A(object o) { }
                 }
+                class B
+                {
+                    object P1 { [A(field)] get { return null; } set { } }
+                    object P2 { get { return null; } [A(field)] set { } }
+                }
                 class C
                 {
-                    object P2 { [A(field)] get { return null; } set { } }
-                    object P3 { get { return null; } [A(field)] set { } }
+                    const object field = null;
+                    object P3 { [A(field)] get { return null; } set { } }
                 }
                 """;
 
             var comp = CreateCompilation(source);
             comp.VerifyEmitDiagnostics(
-                // (8,20): info CS9258: 'field' is a contextual keyword in property accessors starting in language version preview. Use '@field' instead.
-                //     object P2 { [A(field)] get { return null; } set { } }
-                Diagnostic(ErrorCode.INF_IdentifierConflictWithContextualKeyword, "field").WithArguments("field", "preview").WithLocation(8, 20),
                 // (8,20): error CS0182: An attribute argument must be a constant expression, typeof expression or array creation expression of an attribute parameter type
-                //     object P2 { [A(field)] get { return null; } set { } }
+                //     object P1 { [A(field)] get { return null; } set { } }
                 Diagnostic(ErrorCode.ERR_BadAttributeArgument, "field").WithLocation(8, 20),
-                // (9,41): info CS9258: 'field' is a contextual keyword in property accessors starting in language version preview. Use '@field' instead.
-                //     object P3 { get { return null; } [A(field)] set { } }
-                Diagnostic(ErrorCode.INF_IdentifierConflictWithContextualKeyword, "field").WithArguments("field", "preview").WithLocation(9, 41),
                 // (9,41): error CS0182: An attribute argument must be a constant expression, typeof expression or array creation expression of an attribute parameter type
-                //     object P3 { get { return null; } [A(field)] set { } }
-                Diagnostic(ErrorCode.ERR_BadAttributeArgument, "field").WithLocation(9, 41));
+                //     object P2 { get { return null; } [A(field)] set { } }
+                Diagnostic(ErrorCode.ERR_BadAttributeArgument, "field").WithLocation(9, 41),
+                // (14,20): warning CS9258: In language version preview, the 'field' keyword binds to a synthesized backing field for the property. To avoid generating a synthesized backing field, and to refer to the existing member, use 'this.field' or '@field' instead.
+                //     object P3 { [A(field)] get { return null; } set { } }
+                Diagnostic(ErrorCode.WRN_FieldIsAmbiguous, "field").WithArguments("preview").WithLocation(14, 20),
```

Comment:
Isn't our default with no options Preview? And if it isn't, isn't this baseline going to change whenever we increment the default?

Reply:
Yes, the default with no options is language version preview. This baseline will change after the feature ships, with the baseline including the exact language version. I believe this matches how we typically handling language version diagnostics in tests.

Reply:
For those types of baselines, we usually use `RegularNext` as an explicit language version.

Reply:
I've updated most of the tests to use `LanguageVersionFacts.CSharpNext`. I've left this one test unchanged though because I think it should use the latest language version always.